phpstan.neon.dist

```diff
@@ -84,6 +84,10 @@ parameters:
             message: '~^Parameter #1 \$expected of static method PHPUnit\\Framework\\Assert::assertInstanceOf\(\) expects class-string<Doctrine\\DBAL\\Connection&PHPUnit\\Framework\\MockObject\\MockObject>, class-string<Doctrine\\DBAL\\Connection>&class-string<PHPUnit\\Framework\\MockObject\\MockObject> given\.$~'
             path: %currentWorkingDirectory%/tests/DriverManagerTest.php
 
+        -
```

Comment:
I'd file this is an issue with PHPStan (see https://github.com/phpstan/phpstan-mockery/issues/14 for the reference). `iterator_to_array()` obviously may produce side effects.